src/views/Pottery/components/FAQ.tsx

```diff
@@ -0,0 +1,99 @@
+import styled from 'styled-components'
+import { Card, CardBody, CardHeader, Heading, Text, Flex, Link } from '@pancakeswap/uikit'
+import { useTranslation } from 'contexts/Localization'
+import FoldableText from 'components/FoldableSection/FoldableText'
+
+const Wrapper = styled(Flex)`
+  width: 100%;
+  margin: auto;
+  padding: 0 24px 72px 24px;
+  flex-direction: column-reverse;
+  align-items: center;
+
+  ${({ theme }) => theme.mediaQueries.xl} {
+    flex-direction: row;
+    max-width: 1140px;
+  }
+`
+const HoneyImage = styled.div`
+  width: 200px;
+  height: 224.67px;
+  margin: 40px auto auto auto;
+  align-self: flex-start;
+  background: url(/images/pottery/honey.png);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+
+  ${({ theme }) => theme.mediaQueries.xl} {
+    width: 291px;
+    min-width: 291px;
+    height: 326.89px;
+    min-height: 326.89px;
+    margin: 40px 40px 0 0;
+  }
+`
+
+const StyledCardbody = styled(CardBody)`
+  div:first-child {
+    margin-top: 0px;
+  }
+`
+
+const InlineLink = styled(Link)`
+  display: inline-block;
+  margin: 0 4px;
+`
+
+const FAQ = () => {
+  const { t } = useTranslation()
+
+  return (
+    <Wrapper>
+      <HoneyImage />
+      <Card>
+        <CardHeader>
+          <Heading color="secondary" scale="lg">
+            {t('FAQ')}
+          </Heading>
+        </CardHeader>
+        <StyledCardbody>
+          <FoldableText title={t('How is odds calculated?')} mt="24px">
+            <Text fontSize="14px" color="textSubtle">
+              {t(
+                "All CAKE deposits will be locked for the same duration, the odds of your address winning the prize pool is simply proportional to your CAKE deposit over the total CAKE deposit of the whole Pottery. Each week, eight (8) addresses are drawn randomly based on their odds. Simply put, if user A deposited 1 CAKE, user B deposited 2 CAKE, then user B's odds of winning is twice of user A's.",
+              )}
+            </Text>
+          </FoldableText>
+          <FoldableText title={t('Is there any risk associated with depositing and playing Pottery?')} mt="24px">
+            <Text fontSize="14px" color="textSubtle">
+              {t('Is there any risk associated with depositing and playing Pottery?')}
+            </Text>
```

Comment:
It repeated with the title.

![image](https://user-images.githubusercontent.com/109973128/182738121-a9beafaf-489b-4be1-82e2-fb7257f2e30f.png)